scripts/aws/cfn/microservice.yml

```diff
@@ -182,11 +226,218 @@ Resources:
         RestApiStageArns: !Join
           - ','
           - - !GetAtt MandateMicroservicePublicWebAPI.Outputs.RestApiStageArn
+        LambdaArns: !Join
+          - ','
+          - - !GetAtt PnMandateExpiredTriggerLambda.Arn
         RestApiNames: !Join
           - ','
           - - !GetAtt MandateMicroservicePublicWebAPI.Outputs.RestApiName
         AlarmArns: !Join
           - ','
           - - !GetAtt MandateMicroservice.Outputs.FatalLogsMetricAlarmArn
             - !GetAtt MandateMicroservicePublicWebAPI.Outputs.RestApiErrorAlarmArn
-            - !GetAtt MandateMicroservicePublicWebAPI.Outputs.RestApiLatencyAlarmArn
\ No newline at end of file
+            - !GetAtt MandateMicroservicePublicWebAPI.Outputs.RestApiLatencyAlarmArn
+            - !Ref PnMandateExpiredTriggerDLQAlarmARN
+            - !Ref PnMandateInputsQueueAlarmARN
+
```

Comment:
@fabriziolana aggiungere `PnMandateInputsQueueAgeAlarmARN` (vedi commento in basso sullo storage)

Reply:
fatto